test/parallel/test-internal-fs-syncwritestream.js

```diff
@@ -66,6 +66,30 @@ const filename = tmpdir.resolve('sync-write-stream.txt');
   assert.strictEqual(stream.fd, null);
 }
 
+// Verify that the callback will be called when already destroy()ed.
+{
+  const fd = fs.openSync(filename, 'w');
+  const stream = new SyncWriteStream(fd);
+  const theErr = new Error('my error');
+  const cb = () => {};
+
+  stream.on('close', common.mustCall());
+  assert.strictEqual(stream.destroy(), stream);
+  stream.destroy(theErr, common.mustCall(cb));
+  stream.destroySoon(theErr, common.mustCall(cb));
+  stream._destroy(theErr, common.mustCall(cb));
```

Comment:
`writable._destroy()` should not be called directly.

Reply:
Thank you for your comment!
Actually, Once `stream.destroy()` called, the state of `stream` is being updated to be "closed"(here, I think : https://github.com/nodejs/node/blob/main/lib/internal/streams/destroy.js#L105). Therefore, it will be closed regardless of the SyncWriteStream's autoClose property. 
I was trying to do a direct, artificial test rather than modifying this behavior. Do you think such a test would be meaningless? 
For the reasons I mentioned earlier, I'm not sure there is a way to test for `autoClose=false` without directly using `SyncWriteStream.prototype._destroy()`.

Reply:
The way you are currently using `SyncWriteStream.prototype._destroy()` does not test the `autoClose` behavior. It only tests that the provided callback is called. To test the `autoClose` behavior you should verify that `stream.fd` is not closed after calling `stream.destroy()`.

Reply:
Oh, I misunderstood your comment as being about the 2nd test code (about - already destroy()ed)
thanks
I'll reflect your comment soon